docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+data:
+    image: busybox
+    volumes:
+        - ../:/var/www/app
+
+nginx:
+    image: yappabe/nginx
```

Comment:
To use offical nginx image you do not need to define custom dockerfile, basically you need to use one of offical php-fpm images and define own default.conf.

Official nginx image is better cause you can control version of nginx then. This image does not have any tags as well, so I would not use in my own project ;)